src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -455,6 +455,35 @@ public String toString() {
 }
 
 
+enum SensitivityRank {
+    NOT_DEFINED(-1),
+    NONE(0),
+    LOW(10),
+    MEDIUM(20),
+    HIGH(30),
+    CRITICAL(40);
+
+    private int rank;
+
+    private SensitivityRank(int rank) {
+        this.rank = rank;
+    }
+
+    public int getValue() {
+        return rank;
+    }
+
+    static boolean isValid(int rank) throws SQLServerException {
+        for (SensitivityRank r : SensitivityRank.values()) {
```

Comment:
Make values() static array for better performance. See https://github.com/microsoft/mssql-jdbc/pull/1065 for reference.